pyClient/README.md

```diff
@@ -45,11 +55,192 @@ and fails to fetch it (or fails to fetch the right version).
 
 ```console
 # Download the solidity compiler to compile the contracts
-$ wget https://github.com/ethereum/solidity/releases/download/[solc-version]/[solc-for-your-distribution] -O $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
+$ wget https://github.com/ethereum/solidity/releases/download/[solc-version]/[solc-for-your-distribution] \
+    -O $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
 $ chmod +x $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
 ```
 
-To run this command you need to replace the solidity version (denoted by
-`[solc-version]`), the python version (denoted `[python-version]`), and binary
-file (denoted by `[solc-for-your-distribution]`) by your system specific pieces
-of information.
+To run this command, replace the solidity version (denoted by `[solc-version]`),
+the python version (denoted by `[python-version]`), and binary file (denoted by
+`[solc-for-your-distribution]`) by your system specific information.
+
+# The `zeth` command
+
+The `zeth` command exposes Zeth operations via a command line interface.  A
+brief description is given in this section.  More details are available via
+`zeth --help`, and example usage can be seen in the [pyclient test
+script](../scripts/test_zeth_cli).
+
+## Environment
+
+Depending on the operation being performed, the `zeth` client must:
+- interact with an Ethereum RPC host
+- interact with the deployed Zeth contracts
+- request proofs and keys from the `prover_server`, and
+- access secret and public data for the current user
+
+Some of this data could not easily be specified by command line, so the `zeth`
+tools assume it can be found in specific files (with default file names,
+overridable on the command line).
+
+The set of files required for by zeth for a single user is described below.  We
```

Comment:
`required for by zeth for a` -> required by zeth for a

Reply:
Removed